config/StoredProcedures/EditUserByUsername

```diff
@@ -1,14 +0,0 @@
-DROP PROCEDURE IF EXISTS EditUserByUsername;
-
-DELIMITER $$
-CREATE PROCEDURE EditUserByUsername
-(IN _username varchar(255), IN _firstname varchar(255), IN _lastname varchar(255), IN _permissions int, IN _confirmation bit)
-
-BEGIN
-
-UPDATE Users
-SET FirstName = _firstname, LastName = _lastname, PermsID = _permissions, IsConfirmed = _confirmation
-WHERE Username = _username;
-END $$
-
-DELIMITER ;
```

Comment:
This procedure was setting the confirmation bit to 1 every time a "save" occurred and a 0 everytime a delete/deactivation occurred. We need better behavior, so the new version does a check before it touches the bit.